Update and rename github-integration.md to git-integration.md (#2819)
cloudflare/cloudflare-docs: products/pages/src/content/platform/git-integration.md

## Organizational access

> When authorizing Cloudflare Pages to access your GitHub account, you can allow access to organizations/groups that you belong to on GitHub/GitLab. This means that you can deploy projects to Cloudflare Pages from your open-source team, company, or side project. 

Reviewer: ```suggestion
When authorizing Cloudflare Pages to access your GitHub account, you can allow access to organizations/groups that you belong to on GitHub/GitLab. This means that you can deploy projects to Cloudflare Pages from your open-source team, company, or side project.
```

Reviewer: Are we adding "/groups" because that's the official GitLab terminology? Or why the slash?

Reviewer 2: @deadlypants1973 Yes, groups are a collection of projects in GitLab it works like a folder.

> ## Removing access to your account

Reviewer: ```suggestion
## Removing access to your account

```

In order to remove access to your account, you must do so directly on GitHub ot GitLab. Note that removing access to GitHub/GitLab will also disable new builds, though the last build of your site will continue to be hosted via Cloudflare Pages.

> ### GitHub

Reviewer: ```suggestion
### GitHub

```

> View the [**Applications** page](https://github.com/settings/installations) on GitHub to configure your installation settings. 

Reviewer: ```suggestion
Go to **Settings** > [**Applications**](https://github.com/settings/installations) on GitHub to configure your installation settings. 
```

> ### GitLab
> If using GitLab, visit the Applications page on GitLab by navigating to **Preferences** > **Applications** > and clicking **revoke** on the Cloudflare Pages authorized application. 

Reviewer: ```suggestion
### GitLab

Go to **Applications** on GitLab by navigating to **Preferences** > **Applications** > and selecting **Revoke** on the Cloudflare Pages authorized application. 
```
